src/main/java/lotto/domain/Winning.java

```diff
@@ -0,0 +1,34 @@
+package lotto.domain;
+
+import java.util.HashSet;
+
+public class Winning {
+    private HashSet<Integer> winningSet;
```

Comment:
중복이 포함되지 않는다는것을 표현하기 위해 `winningSet` 으로 표현하신걸까요?
```suggestion
    private HashSet<Integer> winningNumbers;
```